File: src/main/java/magicjinn/beaconaura/BeaconAura.java
package magicjinn.beaconaura;

import magicjinn.beaconaura.config.ModConfig;
import net.fabricmc.api.ModInitializer;

public class BeaconAura implements ModInitializer {
	public static final String MOD_ID = "beacon-aura";

	@Override
	public void onInitialize() {
		ModConfig.load();
	}
}

File: src/main/kotlin/magicjinn/beaconaura/BeaconAura.kt
package magicjinn.beaconaura

import net.fabricmc.api.ModInitializer
import org.slf4j.LoggerFactory

object BeaconAura : ModInitializer {
    private val logger = LoggerFactory.getLogger("beacon-aura")

	override fun onInitialize() {
		logger.info("Beacon Aura initialized!")
	}
}